### ISSUE TITLE
Howl#once('loaderror', ...) doesn't work

### ISSUE THREAD

**Issue by AndyBarron** [goldfire/howler.js#670]
Setting the `onloaderror` config property works, but calling `on` or `once` for `'loaderror'` doesn't for some reason.

**Comment by goldfire** [comment]
Can you give sample code? I've just tested with both `on` and `once` and both worked for me.

**Comment by jneuendorf** [comment]
I currently have a similar error. Maybe it just happens in combination with `load`.
I'm just getting started with howler and tried to load a file from `file://` which doesn't work because of cross-origin policies.
Thus there is an error from `XMLHttpRequest` but the below code logs `loaded`.

```javascript
howl.once('loaderror', function() {
  console.log('error') // doesn't happen
}).once('load', function() {
  console.log('loaded') // happens
})
howl.load() // because set preload to false
```

Howler v2.0.4

**Comment by alvisisme** [comment]
The audio element's error event may be binded before on('loaderror') api is called when autoplay option is true.
```
let audio = new Howl({
  html5: true,
  src: 'audio/not_exist.mp3', // the audio file not exists, should trigger error event
  autoplay: true
});
audio.on('load', function () {
  console.log('==>loaded');
});
/**
 * Other events here. 
 * The delay time is uncertain.
 */
// may run after error event binded
audio.on('loaderror', function (msg) {
  console.log('==>loaderror ' + msg);
});
```

set the delay time to test
```
let audio = new Howl({
  html5: true,
  src: 'audio/not_exist.mp3', // the audio file not exists, should trigger error event
  autoplay: true
});
// just test the delay time
setTimeout(function () {
  console.log('loaderror event listened');
  audio.on('loaderror', function (msg) {
    console.log('==>loaderror ' + msg);
  });
  // 10ms is good to catch the error message in my browser but 1000ms is bad
}, 1000);
```

**Comment by ibc** [comment]
Yes, confirmed that when running `howl.play(); howl.on('loaderror', fn)`, the "loaderror" event is never fired, even if the `src` value points to a file that does not exist and `autoplay` is set to `false`.

Any news about this bug?

**Comment by goldfire** [comment]
Closing due to inactivity. If this is still an issue, reply with more info and I'll reopen.

**Comment by ibc** [comment]
Inactivity, yes, because the report was ignored all this time.

**Comment by goldfire** [comment]
> Inactivity, yes, because the report was ignored all this time.

I've put thousands of hours of work into this open source project. Sorry that my free labor is not sufficient enough for you.

**Comment by ibc** [comment]
Don't take me wrong. I just meant that this issue does exist and the reports above confirm it. Closing it due to inactivity from the maintainer does not look much legit to me.

**Comment by goldfire** [comment]
I'm always happy to accept pull requests. The code is open source after all.
